articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -146,12 +145,10 @@ The Call Automation events are sent to the web hook callback URI specified when
 | AddParticipantSucceeded| Your application added a participant  |
 |AddParticipantFailed   | Your application was unable to add a participant  |
 | ParticipantUpdated    | The status of a participant changed while your application’s call leg was connected to a call  |
-| PlayCompleted | Your application successfully played the audio file provided |
-| PlayFailed | Your application failed to play audio |
-| PlayCanceled | Your application canceled the play operation |
+| PlayCompleted| Your application successfully played the audio file provided |
+| PlayFailed| Your application failed to play audio |
 | RecognizeCompleted | Recognition of user input was successfully completed |
 | RecognizeFailed | Recognition of user input was unsuccessful <br/>*to learn more about recognize action events view our how-to guide for [gathering user input](../../how-tos/call-automation/recognize-action.md)*|
-| RecognizeCanceled | Your application canceled the request to recognize user input |
```

Comment:
not sure whats the change here - is this line being removed? Same for PlayCanceled